server/api/tso_test.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2021 TiKV Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package api
+
+import (
+	"fmt"
+	"time"
+
+	. "github.com/pingcap/check"
+	"github.com/tikv/pd/pkg/testutil"
+	"github.com/tikv/pd/server"
+	"github.com/tikv/pd/server/config"
+)
+
+var _ = Suite(&testTsoSuite{})
+
+type testTsoSuite struct {
+	svr       *server.Server
+	cleanup   cleanUpFunc
+	urlPrefix string
+}
+
+func (s *testTsoSuite) SetUpSuite(c *C) {
+	s.svr, s.cleanup = mustNewServer(c, func(cfg *config.Config) {
+		cfg.LocalTSO.EnableLocalTSO = true
+		cfg.LocalTSO.DCLocation = "dc-1"
+	})
+	mustWaitLeader(c, []*server.Server{s.svr})
+
+	addr := s.svr.GetAddr()
+	s.urlPrefix = fmt.Sprintf("%s%s/api/v1", addr, apiPrefix)
+}
+
+func (s *testTsoSuite) TearDownSuite(c *C) {
+	s.cleanup()
+}
+
+func (s *testTsoSuite) TestTransferAllocator(c *C) {
```

Comment:
I think you can add a comment to tell that this is only used to test the API behavior. The real transfer test is at the TSO test part.